docs/index.md

```diff
@@ -13,19 +13,21 @@
 
 CleanRL is a Deep Reinforcement Learning library that provides high-quality single-file implementation with research-friendly features. The implementation is clean and simple, yet we can scale it to run thousands of experiments using AWS Batch. The highlight features of CleanRL are:
 
-
-* Single-file Implementation
-    * **Every detail about an algorithm is put into the algorithm's own file.** Therefore, it's easier for you to fully understand an algorithm and do research with it.
-* Benchmarked Implementation on 7+ algorithms and 34+ games 
-* Tensorboard Logging
-* Local Reproducibility via Seeding
-* Videos of Gameplay Capturing
-* Experiment Management with [Weights and Biases](https://wandb.ai/site)
-* Cloud Integration with Docker and AWS 
+* 📜 Single-file implementation
+   * *Every detail about an algorithm variant is put into a single standalone file.* 
+   * For example, our `ppo_atari.py` only has 340 lines of code but contains all implementation details on how PPO works with Atari games, so it is a great reference implementation to read for folks who do not wish to read an entire modular library.
+* 📊 Benchmarked Implementation (7+ algorithms and 34+ games at https://benchmark.cleanrl.dev)
+* 📈 Tensorboard Logging
+* 🪛 Local Reproducibility via Seeding
+* 🎮 Videos of Gameplay Capturing
+* 🧫 Experiment Management with [Weights and Biases](https://wandb.ai/site)
+* 💸 Cloud Integration with docker and AWS 
 
 You can read more about CleanRL in our [technical paper](https://arxiv.org/abs/2111.08819) and [documentation](https://docs.cleanrl.dev/).
 
-Good luck have fun 🚀
+Good luck have fun :rocket:
+
+⚠️ **NOTE**: CleanRL is *not* a modular library and therefore it is not meant to be imported. At the cost of duplicate code, we make all implementation details of a DRL algorithm variant easy to understand, so CleanRL comes with its own pros and cons. You should consider using CleanRL if you want to 1) understand all implementation details of an algorithm's varaint or 2) do quick prototypes.
```

Comment:
varaint -> variant

I'm not sure that "do quick prototypes" makes sense here. Running `from clearrl import PPO` would be quick. Reading the algorithm and copy-pasting it into my code is slow.

Reply:
I think "doing prototypes" is not really a well-defined notion as there are many types of prototypes. Being able to do prototypes quickly largely depends on the use case. 

While things like `from stable_baselines3 import PPO` is quick but if you want to prototype advanced features that SB3 does not support, it could be more difficult as discussed in #197 with the invalid action masking example.

Maybe I can clarify as "do prototypes that can't be achieved by just combining components in modular DRL libraries"? I am really unsure what the phrasing would be.